app.py

```diff
@@ -36,14 +36,14 @@ def _main():
     args = parser.parse_args()
     # Allow users to pick storage location we may want to
     # store our models in google cloud storage or azure instead of s3 in the future
-    if args.modelstore=="s3":
+    if args.modelstore=="s3" or os.getenv("MODEL_STORE")== "s3":
```

Comment:
Can we use `anomaly_detector.config` to load these environment variables (for consistency)? If not, could you use some prefix in the name (again, for consistency)?

Reply:
What prefix do you recommend @vpavlin ?

Reply:
IIRC we use `LAD_` in the rest of the config

Reply:
adding it now.

Reply:
Fixed it. @vpavlin I'll mark this as resolved. Let me know if there is anything else